tools/uploader.py

```diff
@@ -1,92 +1,31 @@
 #!/usr/bin/env python3
 from bleak import BleakClient, BleakScanner
 from time import sleep
+from ble_ota import uuids
+from ble_ota import consts
+from ble_ota.messages import InitReq, InitResp
+from ble_ota.messages import BeginReq, BeginResp
+from ble_ota.messages import PackageInd, PackageReq, PackageResp
+from ble_ota.messages import EndReq, EndResp
+from ble_ota.messages import SignatureReq, SignatureResp
+from ble_ota.messages import ErrorCode, parse_message_of_type, ERROR_TO_STR
+from ble_ota.utils import get_file_size, is_linux, is_fedora, create_compressed_file, create_signature_file
+from ble_ota.paths import Paths
 import asyncio
 import sys
 import zlib
-import os
 import datetime
-import platform
-
-
-BLE_OTA_SERVICE_UUID = "dac890c2-35a1-11ef-aba0-9b95565f4ffb"
-BLE_OTA_CHARACTERISTIC_UUID_TX = "dac89194-35a1-11ef-aba1-b37714ad9a54"
-BLE_OTA_CHARACTERISTIC_UUID_RX = "dac89266-35a1-11ef-aba2-0f0127bce478"
-BLE_OTA_CHARACTERISTIC_UUID_MF_NAME = "dac89338-35a1-11ef-aba3-8746a2fdea8c"
-BLE_OTA_CHARACTERISTIC_UUID_HW_NAME = "dac89414-35a1-11ef-aba4-7fa301ad5c49"
-BLE_OTA_CHARACTERISTIC_UUID_HW_VER = "dac894e6-35a1-11ef-aba5-0fcd13588409"
-BLE_OTA_CHARACTERISTIC_UUID_SW_NAME = "dac895b8-35a1-11ef-aba6-63ebb073a878"
-BLE_OTA_CHARACTERISTIC_UUID_SW_VER = "dac89694-35a1-11ef-aba7-bf64db99d724"
-
-OK = 0x00
-NOK = 0x01
-INCORRECT_FORMAT = 0x02
-INCORRECT_FIRMWARE_SIZE = 0x03
-CHECKSUM_ERROR = 0x04
-INTERNAL_STORAGE_ERROR = 0x05
-UPLOAD_DISABLED = 0x06
-
-BEGIN = 0x10
-PACKAGE = 0x11
-END = 0x12
-
-respToStr = {
-    NOK: "Not ok",
-    INCORRECT_FORMAT: "Incorrect format",
-    INCORRECT_FIRMWARE_SIZE: "Incorrect firmware size",
-    CHECKSUM_ERROR: "Checksum error",
-    INTERNAL_STORAGE_ERROR: "Internal storage error",
-    UPLOAD_DISABLED: "Upload disabled"
-}
-
-MTU_WRITE_OVERHEAD_BYTES_NUM = 3
-
-U8_BYTES_NUM = 1
-U32_BYTES_NUM = 4
-HEAD_BYTES_NUM = U8_BYTES_NUM
-ATTR_SIZE_BYTES_NUM = U32_BYTES_NUM
-BUFFER_SIZE_BYTES_NUM = U32_BYTES_NUM
-BEGIN_RESP_BYTES_NUM = HEAD_BYTES_NUM + ATTR_SIZE_BYTES_NUM + BUFFER_SIZE_BYTES_NUM
-HEAD_POS = 0
-ATTR_SIZE_POS = HEAD_POS + HEAD_BYTES_NUM
-BUFFER_SIZE_POS = ATTR_SIZE_POS + ATTR_SIZE_BYTES_NUM
-
-
-def file_size(path):
-    if os.path.isfile(path):
-        file_info = os.stat(path)
-        return file_info.st_size
-
-
-def bytes_to_int(value):
-    return int.from_bytes(value, 'little', signed=False)
-
-
-def int_to_u8_bytes(value):
-    return int.to_bytes(value, U8_BYTES_NUM, 'little', signed=False)
-
-
-def int_to_u32_bytes(value):
-    return int.to_bytes(value, U32_BYTES_NUM, 'little', signed=False)
+import os
```

Comment:
```suggestion
import asyncio
import datetime
import os
import sys
from time import sleep
import zlib

from bleak import BleakClient, BleakScanner

from ble_ota import uuids
from ble_ota import consts
from ble_ota.messages import InitReq, InitResp
from ble_ota.messages import BeginReq, BeginResp
from ble_ota.messages import PackageInd, PackageReq, PackageResp
from ble_ota.messages import EndReq, EndResp
from ble_ota.messages import SignatureReq, SignatureResp
from ble_ota.messages import ErrorCode, parse_message_of_type, ERROR_TO_STR
from ble_ota.utils import get_file_size, is_linux, is_fedora, create_compressed_file, create_signature_file
from ble_ota.paths import Paths

```

Reply:
Fixed offline because github print issue and cant apply it